docs/admins/skills.rst

```diff
@@ -0,0 +1,90 @@
+
+Creating a new skill
+--------------------
+
+1. From the **Topics and Skills Dashboard**, click the |createskill| button.
+
+.. |createskill| image:: /images/create_skill.png
+                 :scale: 35 %
+
+2. The **Create New Skill** dialog box appears. Enter a description for the new skill under **New Skill Description**.
+
+3. To enter a rubric for a difficulty level, first select the difficulty level under **Add Skill Rubrics** and then enter the rubric in the text box below.
+
+.. admonition:: Rubrics
+
+   Rubrics help question creators when they create questions for the skill. You can enter rubrics for each difficulty level of the skill (Easy, Medium, Hard) which are then shown to question creators so they know what the question should test. For example, in the image below, if a question creator chooses to create an *easy* question for this new skill, this rubric will be shown as a guide to creating the question:
+
+   .. figure:: /images/new_skill.png
+      :alt: Create new skill
+      :scale: 35 %
+
+      *Fig. 1* 
+
+
+You may enter multiple rubrics by clicking the first |savebutton| button below the text box and selecting the next level of difficulty. 
+
+3. When you are done entering all rubrics, click |savebutton| and you will be directed to the **Skill Editor** screen.
+
+.. |savebutton| image:: /images/save_button.png
+                :scale: 40 %
+
+.. _conceptcard:
+
+Writing the Concept Card for a skill
+--------------------------------------
+
+.. admonition:: Concept Cards
+
+   The **Skill Editor** shows you three cards in addition to the ones mentioned above. The **Review Material** and **Worked Examples** sections are together known as 'Concept Card'. The information you enter here is what a learner will see if they have trouble answering questions related to the skill. In other words, enter information here that will help a learner understand the skill concept better by providing a brief explanation of how to perform that skill, and some examples.
+
+.. admonition:: Misconception card
+
+   The **Misconceptions** card is useful for question creators to ensure that they address all misconceptions when creating questions and answer groups for that skill. The information you enter here will help ensure that answer groups have a sufficient amount of targeted feedback. 
+
+.. figure:: /images/concept_card.png
+   :alt: Concept card
+   :scale: 25 %
+
+   *Fig. 2*
+
+1. When you have entered all the details, to return back to the **Topics and Skills Dashboard** *without* publishing the skill, select the dashboard from the drop-down menu on your username picture in the top-right corner of the screen. This will create a tab called **Unpublished Skills** which lists all skills created and not yet published. 
+
+2. To publish the skill right away, click the |publish| button, and then |publishskill|. This will create a tab called **Unused Skills** which lists all skills published and not yet assigned to any Topic.
```

Comment:
Maybe mention, only a Skill with at least the 'Review Material' field filled can be published?

Reply:
@aks681 At the moment, it looks like you can publish a skill even without the 'Review Material' field filled in. I just tested it and only put in a Skill Description, and I was able to publish the skill. Is this a bug maybe?

Reply:
Oh, yeah, it might be a bug. Will fix that. In the meantime, I think it's fine to edit this?